server/lib/pbench/indexer.py

```diff
@@ -3205,12 +3205,14 @@ def mk_sosreports(tb, extracted_root, logger):
             continue
         host_md = hostnames_if_ip_from_sosreport(os.path.join(extracted_root, sos))
         # get hostname (short and FQDN) from sosreport
-        d = _dict_const()
-        d['name'] = sos
-        d['md5'] = md5_val
-        d.update(host_md)
-        sosreportlist.append(d)
-
+        if 'OK' in host_md:
+            d = _dict_const()
+            d['name'] = sos
+            d['md5'] = md5_val
+            d.update(host_md)
+            sosreportlist.append(d)
+        else:
+            sosreportlist.append(host_md)
```

Comment:
This is the error case: what we do here has to be coordinated with what we want to store in ES so that the dashboard will make sense of it.

Reply:
Will change the value of it after discussion. Currently gave a value which I thought would be suitable.